src/adapter/driver/api/controllers/health/healthController.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from "express";
+import testConnection from "../../../../driven/infra/mongo/helpers/testConnection";
```

Comment:
- dar preferencia para o alias
```suggestion
import testConnection from "@infra/mongo/helpers/testConnection";
```

Reply:
Boa, não conhecia essa funcionalidade do alias